### ISSUE TITLE
End screen- Reward Summary

### ISSUE THREAD

**Issue by ghost** [andrei1058/BedWars1058#206]
Please add this reward summary to bedwars1058's game ending 
https://hypixel.net/attachments/1600890202852-png.2006850/

**Comment by andrei1058** [comment]
Interesting

**Comment by ajgeiss0702** [comment]
The image uploaded here (if hypixel's website blocks your ip like me)
![hyp](https://user-images.githubusercontent.com/6259574/146686648-833032b2-4e39-4d88-a159-de9050794e63.png)

**Comment by ItsFantom** [comment]
oh yeah please do this is good idea :D

**Comment by ghost** [comment]
`$sudo update Bedwars1058 version-code=v41.11.2`
This version will be released on xx-11-2041

**Comment by ghost** [comment]
can i close this now?

**Comment by ajgeiss0702** [comment]
> can i close this now?

No. This will be automatically closed once the PR is merged into the plugin.
